prac_06/guitar.py

```diff
@@ -0,0 +1,18 @@
+CURRENT_YEAR = 2022
+VINTAGE_AGE = 50
+
+
+class Guitar:
+    def __init__(self, name='', year=0, cost=0.0):
+        self.name = name
+        self.year = year
+        self.cost = cost
+
+    def __str__(self):
+        return f'{self.name} ({self.year}) : ${self.cost:.2f}'
```

Comment:
Will this format the cost to match the wanted output with a comma ',' separator?

Reply:
Good catch, I missed that.